libs/mrwebrtc/src/interop/logging_interop.cpp

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// This is a precompiled header, it must be on its own, followed by a blank
+// line, to prevent clang-format from reordering it with other headers.
+#include "pch.h"
+
+#include "callback.h"
+#include "logging_interop.h"
+
+using namespace Microsoft::MixedReality::WebRTC;
+
+namespace {
+
+class InteropLogSink : public rtc::LogSink {
+ public:
+  using MessageCallback = Callback<mrsLogSeverity, const char*>;
+
+  static InteropLogSink* Create(MessageCallback callback) noexcept {
+    try {
```

Comment:
Is the try/catch defensive, or is there something specific we are looking out for?

Reply:
It's defensive against the various `std` methods throwing, since this is a `noexcept` function.